assets/js/trento.jsx

```diff
@@ -116,7 +114,18 @@ function App() {
                     path="hosts/:targetID/executions/last/:checkID/:resultTargetType/:resultTargetName"
                     element={<CheckResultDetailPage targetType={TARGET_HOST} />}
                   />
-                  <Route path="users/new" element={<CreateUserPage />} />
+                  <Route
```

Comment:
I'm not fully understanding what this does, I guess it hides all other routes behind the ForbiddenGuard but why does it need to be a Route element without a path? (sorry for the potentially dumb question :sweat_smile: )

Reply:
Without using the `path`, we implement a layout kind of route:
https://reactrouter.com/en/main/route/route#layout-routes

It applies the guard to the routes that are within this element, and returns the outlet if needed.
It is the same way we protect all the routes with the login, so the not logged users are redirected to the guarded view.